strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/ServerConfigurationController.java

```diff
@@ -249,6 +249,11 @@ public ResponseEntity setServerSettings(@RequestBody ServerSettingsForm serverSe
         configurationManagementService.setPort(serverSettingsForm.getPort());
         configurationManagementService.setInstanceName(serverSettingsForm.getInstanceName());
 
+        if(serverSettingsForm.getMaxFileUploadSizeString() != null){
+            String maxFileUploadSize = serverSettingsForm.getMaxFileUploadSizeString();
+            boolean wasAbleToSet = configurationManagementService.setMaxUploadFileSize(maxFileUploadSize);
+        }
```

Comment:
Cosmetic: Code style...
```suggestion
        if (serverSettingsForm.getMaxFileUploadSizeString() != null)
        {
            String maxFileUploadSize = serverSettingsForm.getMaxFileUploadSizeString();
            boolean wasAbleToSet = configurationManagementService.setMaxUploadFileSize(maxFileUploadSize);
        }
```